### PR TITLE
Add help on fields in lhasa

### PR DESCRIPTION
**Pull request by yroffin**
Add help icon in lhasa, and a default help on apps

See on preview ...

capture d ecran 2018-06-29 a 13 44 25

### CODE REVIEW

File: webui/src/app/widget/help-widget/help-widget.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.can-be-clicked {
+    cursor: pointer;
+}
```
Comment (rayene) [inline webui/src/app/widget/help-widget/help-widget.component.css]: missing newline

File: webui/src/app/widget/help-widget/help-widget.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<i title="help" class="oui-icon oui-icon-info_circle oui-icon_bicolor can-be-clicked" aria-hidden="true" (click)="help.request(key)"></i>
```
Comment (rayene) [inline webui/src/app/widget/help-widget/help-widget.component.html]: missing newline

File: webui/src/app/widget/help-widget/help-widget.component.css (hunk 1 above)
Comment (yroffin) [reply webui/src/app/widget/help-widget/help-widget.component.css]: Done

File: webui/src/app/widget/help-widget/help-widget.component.html (hunk 2 above)
Comment (yroffin) [reply webui/src/app/widget/help-widget/help-widget.component.html]: Done

**Pull request merged**